src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -7658,27 +7613,15 @@ public class FileSystemContentDynamicParametersBase
 #endif
 
         /// <summary>
-        /// Gets the encoding from the specified StreamType parameter.
-        /// </summary>
-        public Encoding EncodingType
-        {
-            get
-            {
-                return Utils.GetEncodingFromEnum(Encoding);
-            }
-        } // EncodingType
-
-        /// <summary>
         /// Gets the Byte Encoding status of the StreamType parameter.  Returns true
         /// if the stream was opened with "Byte" encoding, false otherwise.
         /// </summary>
         public bool UsingByteEncoding
         {
-            get
-            {
-                return Encoding == FileSystemCmdletProviderEncoding.Byte;
-            } // get
+            get { return _usingByteEncoding; }
```

Comment:
Please use automatic property and remove _usingByteEncoding.

Reply:
fixed